src/components/screenings/CategoryFilter.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-select v-model="categoryFilter" class="category-filter" placeholder="Choose category" :options="categoriesOptions"/>
+</template>
+
+<script>
+export default {
+    data() {
+        return {
+            categoryFilter: ''
+        }
+    },
+    computed: {
+        categoriesOptions() {
+            return this.genres.map(genre => ({label: genre.name, code: genre.id}));
+        },
+        genres() {
+            return this.$store.getters.genres;
+        }
+    },
+    watch: {
+        categoryFilter(newVal) {
+            this.$emit("categoryChange", newVal)
+        }
+    }
+}
+</script>
+
+<style lang="scss" scoped>
+.category-filter {
+    max-width: 50%;
+    min-width: 200px;
+    height: 100%;
+    @include sm { 
+        margin-left: 0;
+        max-width: 100%;
+        width: 100%;
+    }
+}
+</style>
+
+<style lang="scss">
+:root {
+    --vs-border-color: inherit;
+    --vs-dropdown-bg: #F7F7F7;
+    --vs-border-radius: 10px;
+    --vs-dropdown-option--active-bg: #EAEAEA;
+    --vs-dropdown-option--active-color: inherit;
+}
```

Comment:
setting styles that are leaking outside of component is bad practice: consider setting it for this component locally e.g.

```suggestion
.category-filter {
    --vs-border-color: inherit;
    --vs-dropdown-bg: #F7F7F7;
    --vs-border-radius: 10px;
    --vs-dropdown-option--active-bg: #EAEAEA;
    --vs-dropdown-option--active-color: inherit;
}
```

or if it won't work, maybe you should create some global styles for it?

Reply:
So i figured out that the way you did above works fine! But... there are more things I need to change. It requires  me to reach element by its class. If I do so it doesn't work because of `scoped`. Im wondering is it ok to leave them here in another not `scoped` style block?

Reply:
@rossinek 🖐️

Reply:
Sorry I missed your comment. Not using scoped is one way to go but fortunately there is even better way! You can use "deep selector" - `::v-deep` should do the trick.

https://vue-loader.vuejs.org/guide/scoped-css.html#child-component-root-elements

Reply:
e.g.
```scss
.v-select {
  // some styles
  ::v-deep .some-inner-element {
    // override styles here
  }
}
```